### ISSUE TITLE
Migrating: Grackle GraphQL

### ISSUE THREAD

**Issue by rossabaker** [typelevel/governance#111]
## Migration Checklist

Project Submission Issue: #110 

### For Organization Projects:

- [ ] Create a team for the repo with maintainer as team member
- [ ] After repo transfer, add repo with write permissions to the created team

### For All Projects:

- [ ] Add project to the site by [adding an entry to the projects.yml file](https://github.com/typelevel/typelevel.github.com/blob/development/_data/projects.yml)

### Announcements

These steps are optional but encouraged.

- [ ] Write a blog post announcing the change
- [ ] Post to our social channels (Mastodon)

**Comment by rossabaker** [comment]
Created the @typelevel/grackle team with @milessabin as initial member.  Let me know if there are any comaintainers.  We'll get it attached to the repo after transfer.

**Comment by milessabin** [comment]
Could you add @tpolecat, @dantb and @jf-botto as comaintainers. Thanks!

**Comment by rossabaker** [comment]
Invites sent.  

Is the plan to transfer the repo, rather than fork it into the org?

**Comment by armanbilge** [comment]
Better to transfer if possible, then it carries issues/PRs/etc.

**Comment by milessabin** [comment]
Yes, the plan is to transfer. @tpolecat will be able handle things on the Gemini end.

**Comment by milessabin** [comment]
Could you also add @camimora ... or is there some way I can do that?

**Comment by rossabaker** [comment]
Invite sent.  Happy to add any more, but we're at our GitHub seat cap, and will need to retire some dormant members and/or buy seats before any additional.

**Comment by reardonj** [comment]
This is on the project page, and I see it's being published under the Typelevel org.
